src/common/tamq/tamq_sub.cpp

```diff
@@ -64,17 +64,16 @@ TAMQ_Consumer::TAMQ_Consumer( const std::string& brokerURI,
     {
         LOG_VERBOSE(4, "TAMQ Broker Address: %s",   brokerURI.c_str());
         LOG_VERBOSE(4, "TAMQ Command URI: %s",      destURI.c_str());
-        LOG_VERBOSE(4, "TAMQ Use Topics: %s",       useTopic ? "true" : "false");
+        LOG_VERBOSE(4, "TAMQ Use Topics: %s",       useTopic  ? "true" : "false");
```

Comment:
unnecessary white space :)

Reply:
Lol, I added that so that the "true": "false" lines up with the next line. Just looks better to me.
I really hate trailing whitespace because git picks it up as a change, when its not intended to be there at all.